apps/ios/Shared/Views/Chat/Group/AddGroupMembersView.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  AddGroupMembersView.swift
+//  SimpleX (iOS)
+//
+//  Created by JRoberts on 22.07.2022.
+//  Copyright © 2022 SimpleX Chat. All rights reserved.
+//
+
+import SwiftUI
+import SimpleXChat
+
+struct AddGroupMembersView: View {
+    @EnvironmentObject var chatModel: ChatModel
+    var groupId: Int64
+    @Binding var chatViewSheet: ChatViewSheet?
+    @State private var unaddedContacts: [Contact] = []
+    @State private var selectedContactIds = Set<Int64>()
+
+    var body: some View {
+        VStack {
+            Text("Contacts")
+                .font(.headline)
+                .padding()
+            if (unaddedContacts.isEmpty) {
+                Text("No unadded contacts")
+                    .foregroundColor(.secondary)
+            } else {
+                HStack {
+                    Button {
+                        selectedContactIds.forEach { contactId in
+                            Task {
+                                await addMember(groupId: groupId, contactId: contactId)
+                            }
+                            chatViewSheet = nil
+                        }
+                    } label: {
+                        Label("Add members", systemImage: "plus.circle.fill")
+                    }
+                    .padding()
+                    Spacer()
```

Comment:
do not use Spacer for alignment like that - it should be alignment parameter in HStack